lib/ecto/schema.ex

```diff
@@ -422,7 +427,9 @@ defmodule Ecto.Schema do
     * `:default` - Sets the default value on the schema and the struct.
       The default value is calculated at compilation time, so don't use
       expressions like `DateTime.utc_now` or `Ecto.UUID.generate` as
-      they would then be the same for all records
+      they would then be the same for all records.
+
+    * `:source` - Defines the name that is to be used in database for this field.
```

Comment:
@michalmuskala - Yes, I agree. The `source` should be defined as an atom. However, I am not able to understand what needs change as per this comment?